.gitignore

```diff
@@ -1,6 +1,8 @@
-__pycache__/
 .env
-data/
-backend/src/source_list.json
-.ruff_cache
-backend/src/secret.json
\ No newline at end of file
+__pycache__/
+*.pyc
+frontend/secrets.json  
+evaluation/human_evaluation/secrets.json  
+secrets.json
```

Comment:
This covers all instances of `secrets.json` in the repo.

Reply:
Yes, it does. However, people using some other naming scheme can go under the radar as while exporting the credentials from gcp, it's usually <PROJECT_ID>.json

Reply:
The practice is either you leave the last line to cover everything or just add in every instance of `secrets.json`. Not both.

Reply:
If there is need for that <PROJECT_ID>.json, do `human_evaluation/*.json` or something

Reply:
Yeah, I think this approach will be be better.

> If there is need for that <PROJECT_ID>.json, do `human_evaluation/*.json` or something